code/src/components/PostList.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-underscore-dangle */
+import React from 'react';
+import moment from 'moment';
+
+const PostList = ({ postList, onLikesIncrease }) => {
+  return (
+    <section className="postList-wrapper">
+      {postList.map((thoughtMessage) => {
+        return (
+          <div className="post-wrapper" key={thoughtMessage._id}>
+            <p>{thoughtMessage.message}</p>
+            <div className="post-gutter">
+              <div className="like-button">
+                <button
+                  type="button"
+                  onClick={() => onLikesIncrease(thoughtMessage._id)}
```

Comment:
Like button works well and doesn't glitch when pressed, the experience is smooth